NovelRT.Sdk/TemplateFiles/CMakeTemplate/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(###PROJECT_NAME###_NOVELRT_VERSION ###NOVELRT_VERSION### CACHE STRING "Novel
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 set(GAME_RESOURCES "${CMAKE_SOURCE_DIR}/Resources")
 
-include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)

```

Comment:
This is to go along with the `conanfile.py` PR as the generator switches here. However, if you find that this does wonky things to Ninja, let me know and we can change this back.